javascript/javascript2/week1/week1Js2.js

```diff
@@ -0,0 +1,107 @@
+
+// shortest Word;
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shorterword(danishWords) {
+    let shortestWord;
+    for (let i = 0; i < danishWords.length; i++) {
+        if (danishWords[i].length < danishWords[i++].length) {
+            shortestWord = danishWords[i];
+        }
+        else {
+            shortestWord = danishWords[i++]
+        }
+    }
+    return shortestWord;
+} 
+
+
+console.log(shorterword(danishWords))  // returns 'ø'
```

Comment:
As the basic requirement , this output need to be printed in HTML using DOM manipulation.